package.json

```diff
@@ -24,21 +24,18 @@
     "index.js"
   ],
   "devDependencies": {
-    "babel": "^6.1.18",
-    "babel-preset-es2015": "^6.1.18",
-    "babel-tape-runner": "^2.0.0",
+    "ava": "^0.11.0",
```

Comment:
I removed the `node-Modules` folder from the AppVeyor cache and:

WordPress' currently supports "build tools" on each of these NodeJS versions, as such I'd like to try and keep compatibility of all these NodeJS versions for both *nix and Windows.